src/api/app/views/webui/request/_actions_details.html.haml

```diff
@@ -19,7 +19,7 @@
                                                                         request_action_id: action_item.id,
                                                                         full_diff: diff_limit,
                                                                         diff_to_superseded: diff_to_superseded_id),
-                                                      class: "dropdown-item #{action_item.id == active_action.id ? 'disabled' : ''}")
+                                                      class: "dropdown-item #{action_item.id == active_action.id ? 'active' : ''}")
```

Comment:
This change is also introduced in https://github.com/openSUSE/open-build-service/pull/13681.
Let's keep an eye on the conflicts.

Reply:
There's an extra change in https://github.com/openSUSE/open-build-service/pull/13681/commits/606293751399a8a9910e919eb0c41dc7c3867bf0 with `.border-top` on the `<li>` element. The commit message/description are also more descriptive of what was done and why it was done, instead of how the changes were done.